src/index.html

```diff
@@ -6,13 +6,60 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="photo"
+        class="container__photo"
+      />
+      <h3 class="container__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="container__code">Product code: 195434</p>
+      <div class="container__feedback">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <article>Reviews: 5</article>
+      </div>
+      <div class="container__price">
+        <p class="container__price container__price--price">Price:</p>
+        <p class="container__price container__price--sum">$2,199</p>
```

Comment:
The class names you've used here are a bit confusing. It seems like you're using BEM methodology for naming your classes, which is great. But in BEM, each element should have a unique class, and modifiers are used to signify variations of a block or an element. Here, you've used the same class for two different elements (`container__price`), and it looks like `container__price--price` and `container__price--sum` are meant to be modifiers, but they're not modifying `container__price`, they're separate elements. Consider renaming these classes to make their purpose more clear.